src/content/troubleshooting/_index.en.md

```diff
@@ -0,0 +1,136 @@
++++
+title = "Troubleshooting Guide"
+date = 2020-05-04T19:01:14+05:30
+weight = 5
+pre = "<b>1. </b>"
++++
+
+## Overview
+
+You have folowed steps in [Deployment](../deployment) but something has gone wrong. You're not sure what and how to fix it, or what information to collect to raise an issue. Welcome to the Submariner troubleshooting guide where we will help you get your deployment working again.
+
+To recap, Submariner consists of two main core components (broker and submariner) and optional components (lighthouse), detailed in [Architecture](../architecture) section. Basic familiarity with the architecture will be helpful when troubleshooting.
+
+The guide has been broken into different sections for easy navigation.
+
+### Pre-requisite
+Before we begin troubleshooting, it would be good to know which version of submariner and its components you are running.
+
+{{% notice info %}}
+
+TBD: Add information on how to find out which version users are running. Is `subctl version` enough?
+
+{{% /notice %}}
+
+Get information about the service you're trying to access.
+
+`kubectl get services |grep <service-name` will provide you with Service *Name*, *Namespace* and *ServiceIP*. If you enabled **Globalnet** you will also need the *GlobalnetIP* of the service by running
+
+``` kubectl get service <service-name> -o jsonpath='{.metadata.annotations.submariner\.io/globalIp}' ```
+
+### Deployment Issues
+This section will contain information about common deployment issues you can run into.
+
+#### TBD
+
+### Connectivity Issues
+Deployment went through successfully but services/pods on one cluster are unable to connect to services in another clusters. This can be due to multuple factors - IP Sec tunnels, IP Table rules, Gateway pods, Routeagent etc.
```

Comment:
```suggestion
Deployment went through successfully but services/pods on one cluster are unable to connect to services on another clusters. This can be due to multuple factors - IP Sec tunnels, IP Table rules, Gateway pods, Routeagent etc.
```